### ISSUE TITLE
[FR] Pas de score dans le tableau pour un joueur

### ISSUE THREAD

**Issue by ramok0** [UpGunGame/UpGun-Issues#455]
**Version/Mode de jeu**
2.0.5.0/Free for All

**Décrivez le bug**
Un joueur qui a rejoint la partie manche 2 n'a pas de score affiché sur le scoreboard, l'host ne peut pas le voir, le joueur qui a rejoint non plus, mais un autre joueur peut le voir
Inutile de préciser que le joueur qui n'avait pas de score avait évidemment fait des kills etc, je pense que c'est juste un bug d'affichage

**Pour reproduire le bug**
Etapes pour reproduire le bug (je ne sais pas si c'est a tout les coups):
1. Démarrer la partie
2. Jouer la partie
3. Relancer une partie
4. Jouer une manche
5. Faire rejoindre un joueur
6. Le joueur sera toujours marqué comme un specteur

**Comportement attendu**
Le joueur devrait avoir un score affiché dans le tableau

**Screenshots**
![image](https://github.com/UpGunGame/UpGun-Issues/assets/63268168/efd5a184-5e51-4249-b616-b978544b0299)
![image](https://github.com/UpGunGame/UpGun-Issues/assets/63268168/d53ae326-a4bb-42a2-9b9a-1fe7b70cb0d7)

**Configuration de votre PC:**
 - Système d'exploitation: Windows 10 22H2
 - Processeur: AMD Ryzen 7 5700X
 - RAM: 48gb 3200mhz
 - Carte graphique: rtx 3060

**Comment by MaxEstDev** [comment]
C'est pas un bug, c'est un spectateur nan ?
Au vu de sa couleur.

**Comment by ramok0** [comment]
Non, il jouait ingame avec nous, de la manche 1 jusqu'a la manche 5, il était même sur le podium
Par contre oui effectivement a la manche 1 il était spectateur

**Comment by ramok0** [comment]
Et c'est justement ça que je trouve bizzare, étant donné que j'étais host et que donc j'avais reçu le packet pour changer d'équipe, je vois pas comment une partie du jeu pourrait penser que le mec est spectateur et une autre faire comme si de rien était et laisser le mec jouer tranquillement

**Comment by ramok0** [comment]
Je confirme que c'est systématique quand quelqu'un join la partie pendant qu'elle est déjà en cours, Flex a rejoint en plein milieu de la game, il avait aucune stats dans le hud, mais la game d'après c'est revenu

Donc comment reproduire : 

- Démarrer une partie avec au moins 2 joueurs
- Faire rejoindre une personne

https://streamable.com/b4rzux => Flex peut jouer mais aucune stats dans mon HUD
HUD de la partie d'après (comportement attendu) : 
![image](https://github.com/UpGunGame/UpGun-Issues/assets/63268168/55b971d8-ec26-400e-9e5c-99e61f05016b)
